lighthouse-core/test/config/config-test.js

```diff
@@ -365,53 +370,51 @@ describe('Config', () => {
     });
   });
 
-  describe('getAggregations', () => {
+  describe('getCategories', () => {
     it('returns the IDs & names of the aggregations', () => {
-      const runConfig = JSON.parse(JSON.stringify(defaultConfig));
-      const aggs = Config.getAggregations(runConfig);
-      assert.equal(Array.isArray(aggs), true);
-      assert.equal(aggs.length, 5, 'Found the correct number of aggregations');
-      const haveName = aggs.every(agg => agg.name.length);
-      const haveID = aggs.every(agg => agg.id.length);
-      assert.equal(haveName === haveID === true, true, 'they dont have IDs and names');
+      const categories = Config.getCategories(origConfig);
+      assert.equal(Array.isArray(categories), true);
+      assert.equal(categories.length, 4, 'Found the correct number of categories');
+      const haveName = categories.every(agg => agg.name.length);
+      const haveID = categories.every(agg => agg.id.length);
+      assert.equal(haveName === haveID === true, true, 'they have IDs and names');
     });
   });
 
-  describe('generateConfigOfAggregations', () => {
-    const aggregationIDs = ['perf'];
-    const totalAuditCount = defaultConfig.audits.length;
-
+  describe('generateConfigOfCategories', () => {
     it('should not mutate the original config', () => {
-      const origConfig = JSON.parse(JSON.stringify(defaultConfig));
-      Config.generateNewConfigOfAggregations(defaultConfig, aggregationIDs);
-      assert.deepStrictEqual(origConfig, defaultConfig, 'Original config mutated');
+      const configCopy = JSON.parse(JSON.stringify(origConfig));
+      Config.generateNewConfigOfCategories(configCopy, ['performance']);
```

Comment:
just as an FYI, the rename of "perf" to "performance" will break any saved checkboxes people have in audits 2.0.
(that's totally fine with me at this stage, but something to keep in mind going forward)